docs/results/resultinternalforce2dedge.md

```diff
@@ -0,0 +1,54 @@
+# ResultInternalForce2DEdge
+
+**Internal force on 2D edge**
+
+Internal forces on edge of 2D member. The coordinate system of the results is the coordinate system of the member the results belong to.
+
+```{image} ../.gitbook/assets/48_resultinternalforce2dedge_1.gif
+:width: 400px
+```
+
+## Specification in the excel
+
+| Column Header | Data Type | Example / enum definition | Required | Description |
+|---|---|---|---|---|
+| Result on | Enum | On edge | yes | Specify where the result is, only option is 'on edge'. Prepared for future expansion (on opening edge..)  |
+| 2D member | String | B1 | yes | Reference to the name of 2D member in [StructuralSurfaceMember](../structural-analysis-elements/structuralsurfacemember.md) |
+| Edge | Integer | 2 | yes | The index starting with 1. The order is according to order of “edges” property in [StructuralSurfaceMember](../structural-analysis-elements/structuralsurfacemember.md) |
+| Result for | Enum | Load case<br>Load combination | yes | Specifies from where the result is coming from (from Load Case, Load Combination) |
+| Load case | String | LC1 | yes, if Result for = Load case | Reference to the name of [StructuralLoadCase](../loads/structuralloadcase.md) |
+| Load combination | String | COM1 | yes, if Result for = Combination | Reference to the name of [StructuralLoadCombination](../loads/structuralloadcombination.md) |
+| Combination key | String | 1.35LC1+1.5LC2+1.5LC3+1.5LC4 | no | Allows to define exact combination per result section<br><br>Structure of string:<br>”LoadFactor1LoadCase1+LoadFactor2LoadCase2<br>+LoadFactorN*LoadCaseN”<br>For envelopes and national standard (code) combinations, this column specifies for which exact combination the result is computed |
+| Section at [m] | Double | 0.100 | yes | X coordinate on the edge(distance from the start node) where the result is located |
+| Index | Integer | 1 | yes | Index of the section on edge |
+| mx [kNm/m] | Double | 3.00 | yes | Result value of bending moment mx |
+| my [kNm/m] | Double | 3.00 | yes | Result value of bending moment my |
+| mxy [kNm/m] | Double | 3.00 | yes | Result value of torsion moment mxy |
+| vx [kN/m] | Double | 3.00 | yes | Result value of shear force vx |
+| vy [kN/m] | Double | 3.00 | yes | Result value of shear force vy |
+| nx [kN/m] | Double | 3.00 | yes | Result value of membrane force nx |
+| ny [kN/m] | Double | 3.00 | yes | Result value of membrane force ny |
+| nxy [kN/m] | Double | 3.00 | yes | Result value of shear force nxy |
+
+## Notes
+
+>### Multiple tables in one sheet
+>
+>The amount of data can be limited due to the [limitation of xlsx](https://support.microsoft.com/en-us/office/excel-specifications-and-limits-1672b34d-7043-467e-8e27-269d656771c3) format.
+>
+>Therefore the results can be written to SAF in a form of multiple tables. In the similar logic as the [StructuralProxyElement](../structural-analysis-elements/structuralproxyelement.md).
```

Comment:
The only remark I have for this line is that the page for StructuralProxyElement doesn't specify either what "Multiple tables" should be.

It's nowhere to be found in the docs what exactly this means...

Reply:
Yeah it is not clear, I created an issue, that we should specify it there, do you think I should delete it until it is specified?
https://github.com/StructuralAnalysisFormat/StructuralAnalysisFormat-Doc/issues/74

Reply:
I would suggest leaving it there for consistency (there is same note under 1D results). And fix the issue before release.

Reply:
Maybe add a new Annex page describing the different sheet types?

* Worksheet that describes a single object type:
  * Always 2 columns:
    * First column contains the property names
    * Second column contains the property values
* Worksheet that describes multiple objects of a single type:
  * Always has X columns, X being the amount of properties on that object type
  * Each row represents an entire object
  * Allows to define 1,048,575 objects
* Worksheet that describes multiple objects of a single type and at the same time circumvents Excel's limitations:
  * Sheet is divided into "tables", where each table has X amount of columns, one for each property of the object type
  * Each row represents an object
  * If the maximum row count for the sheet is reached, a new table is started at the top on the same sheet, leaving a blanc column between the tables.
 Might be nice to show an example (eg StructuralProxyElementVertex)
  * The amount of tables allowed on the sheet depends on the amount of properties of the object.
 e.g. if the object has 5 properties, each table will have 5 columns.
 Excel allows 16,384 columns on a sheet and we'd need 5+1 columns per table, which means 2730 full tables
 This in turn allows us to define a total of 2,730 x 1,048,575 = 2,862,609,750 objects on the sheet, opposed to "only" 1,048,575 objects if a single table was used.

Reply:
I like it :) I will copy the idea into the issue and discuss it internally. 
I deleted the link as (if we create a new page) it might be different.

Reply:
Alright, fine with me